digitiser-aggregator/src/frame/cache.rs

```diff
@@ -82,7 +83,7 @@ where
         if self
             .frames
             .front()
-            .is_some_and(|frame| frame.is_complete(&self.expected_digitisers) | frame.is_expired())
+            .is_some_and(|frame| frame.is_complete() | frame.is_expired())
```

Comment:
Prefer the logical operators when doing logical operations.

```suggestion
            .is_some_and(|frame| frame.is_complete() || frame.is_expired())
```